### PR TITLE
Create README.md with API/SPI doc publishing instructions

### PR DESCRIPTION
**Pull request by dmuelle**

### CODE REVIEW

**Comment by dmuelle** [comment]
Hi @mbroz2 @kinueng , here is the draft for the Javadoc publishing instructions. This file covers only the part that is relevant to the docs-javadoc repo. I will update the doc on docs-playbook to point to these steps once we've got them sorted.

one thing I'm not sure about is step 2:

> Get the generated `com.ibm.openliberty.javadoc.doc.jar` file from the Liberty GM build.

I know how to do that, but I'm not sure how much detail we want about it in this readme or if there is any link I should include

**Comment by kinueng** [comment]
Hi @steven1046, do these steps look like the steps you took to deliver the most recent set of API/SPI Javadocs?

File: README.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
+# Open Liberty Javadoc 
+This repository contains the Javadoc that is published on https://openliberty.io, including the following models:
+
+- Open Liberty APIs
+- Open Liberty SPIs
+- MicroProfile APIs
+- Jakarta EE APIs
+- Java EE APIs
+
+Microprofile and Jakarta API documentation is updated with each major release from those platforms. The Open Liberty API and SPI documentation is updated with each 4-week release of the runtime.
```
Comment (steven1046) [inline README.md]: Capitalize the p in MicroProfile

**Review by steven1046 — COMMENTED** [review]
The steps look correct to me. I just had one minor spelling update

File: README.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+1. Clone the `docs-javadoc` repository 
+2. Get the generated `com.ibm.openliberty.javadoc.doc.jar` file from the Liberty GM build.
+3. Change the extension of the file from `com.ibm.openliberty.javadoc.doc.jar` to `com.ibm.openliberty.javadoc.doc.zip`.
+4. Extract the `com.ibm.openliberty.javadoc.doc.zip` to a temporary directory on your local machine.
+5. Create a branch off the `prod` branch of `docs-javadoc`.
+6. In the `modules/reference` directory, create a new folder and name it based on the version for that release, for example `modules/reference/23.0.0.7`.
+7. Copy all the `-javadoc` folders from your local extracted copy of the Javadoc file into the new `modules/reference/<version>` folder that you created in the previous step.
+8. Commit your changes to the branch that you created in step 5.
+9. Create a pull request from your branch to `staging`. Do not add the files to `draft`.
+10. Get the PR reviewed by a docs team member and merge it.
```
Comment (mbroz2) [inline README.md:27]: @cbridgha when 'you' push the docs-generated content, are doing that manually, or is it ran as part of a piece of automation?
It feels like the upload of both the docs-generate and the Liberty API/SPI (steps 1-10 above) could be automated.  From my understanding, the GM automation build already generates some PRs, so I would hope that most of the effort to automate these two tasks is largely already in place?

File: README.md (hunk 1 above)
Comment (dmuelle) [inline README.md]:
```suggestion
MicroProfile and Jakarta API documentation is updated with each major release from those platforms. The Open Liberty API and SPI documentation is updated with each 4-week release of the runtime. 
```

**Review by steven1046 — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (dmuelle) [reply README.md]: thanks, fixed

**Review by mbroz2 — APPROVED** [review]

**Pull request merged**